lib/tasks/npm-install.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
 // Runs `npm install` in cwd
-
-var NpmTask = require('./npm-task');
+var existsSync   = require('exists-sync');
+var chalk        = require('chalk');
+var Promise      = require('../ext/promise');
+var NpmTask      = require('./npm-task');
 
 module.exports = NpmTask.extend({
   command: 'install',
   startProgressMessage: 'Installing packages for tooling via npm',
-  completionMessage: 'Installed packages for tooling via npm.'
+  completionMessage: 'Installed packages for tooling via npm.',
+  run: function(){
+    var ui = this.ui;
+
+    if (!existsSync('./package.json')) {
```

Comment:
I didn't want to expand the scope beyond install failures on `package.json`-less blueprints. I can update to have it warn on all npm commands, if you want.